pincer/middleware/payload.py

```diff
@@ -13,8 +13,9 @@
 
 async def payload_middleware(
     payload: GatewayDispatch
-) -> Tuple[str, GatewayDispatch]:
-    """Invoked when basically anything is received from gateway.
+) -> Tuple[str, List[GatewayDispatch]]:
```

Comment:
why? it returns a tuple of string and `GatewayDispatch`

Reply:
@Sigmanificient

Reply:
This still has to be fixed

Reply:
Only thing I removed was basically

Reply:
you changed it and your local changes overwrote recent changes

Reply:
https://github.com/Pincer-org/Pincer/pull/247/commits/1fd8fb7fcc5c9c9f21a7dd364d81f92af1e37fbb#diff-16cbcd033a37c178ebde6ab43f312808903f179651ee0b5e686b5fc7e196352aR17

Reply:
it's still there when i try to edit it in my IDE

Reply:
[moment](https://cdn.upload.systems/uploads/nfqFqpyU.png)

Reply:
fixed https://github.com/Pincer-org/Pincer/pull/247/commits/1e4914cb667ce5dbc52bce9242f8803a15f6e2bf